test_rclcpp/test/test_multiple_service_calls.cpp

```diff
@@ -34,7 +34,7 @@ void handle_add_two_ints(
   response->sum = request->a + request->b;
 }
 
-TEST(CLASSNAME(test_multiple_service_calls, RMW_IMPLEMENTATION), multiple_service_calls) {
+TEST(CLASSNAME(test_two_service_calls, RMW_IMPLEMENTATION), two_service_calls) {
```

Comment:
Please also update the node and topic names accordingly. Otherwise they collide with the new test.